netdata.spec.in

```diff
@@ -441,48 +436,6 @@ install -m 755 -p packaging/installer/%{name}-updater.sh "${RPM_BUILD_ROOT}%{_li
 install -m 755 -d "${RPM_BUILD_ROOT}%{_sysconfdir}/logrotate.d"
 install -m 644 -p "%{__cmake_builddir}/system/logrotate/%{name}" "${RPM_BUILD_ROOT}%{_sysconfdir}/logrotate.d/%{name}"
 
-# ###########################################################
```

Comment:
@Ferroin Not  100% sure if the following removed code was a fall back mechanism, I will check that don't bother if you are certain

Reply:
All of it is commented out and has been for some time now. We’re not actually running any of it anymore because it was fixing issues in the build system that don’t actually exist since the switch to CMake.